### ISSUE TITLE
[Item/Loot] Set bonus of DK t8 tanking tier

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#3565]
Here it goes, can you finally fix the set bonuses too...

This is how a DK t8 tanking item's SET BONUS must look like:

![tankbonus](https://user-images.githubusercontent.com/31132743/64451524-b20f9f80-d0e4-11e9-8f17-acbc32cc3d35.png)

This is currently the set bonus on 2 of the pieces (ID 46119, 46122) which are wrong item IDs replacing the original intended (46240, 46236) to fix the color and then re-added stats, but you didnt fix the bonus:
![dpsbonus](https://user-images.githubusercontent.com/31132743/64451614-dff4e400-d0e4-11e9-96c6-8391cd2b3215.png)

**How it should work**: DK tank t8 bonus must apply to 2piece=runestrike, 4piece=antimagic shell,as shown on screenshot.
**How it works atm**: 2 of the pieces have the DPS bonus instead, applying to 2piece= deathcoil/frost strike and 4piece=the bonus damage of abilities instead

PS: you can choose to swap back the item IDs and go with wrong color, or fix the set bonus on the new IDs that have the proper skin (and now stats too). I dont care which you do, just dont keep closing issue with leaving broken tier item in game.

**Comment by wow-mania** [comment]
Then use the correct items for the set.

Tank: 46238, 46233, 46234, 46236, 46240
DPS: 46117, 46115, 46111, 46113, 46116

![image](https://user-images.githubusercontent.com/12248683/64471842-1581e780-d14e-11e9-8925-3036de3b1ae7.png)

**Comment by Dranah** [comment]
I would, but sadly those are not the items that are currently in the game. 
Please fix the item IDs on the vendors and Emalon, they are different ones.

**Comment by wow-mania** [comment]
Check the T8 NPC at the mall
If the problem is the other NPC then we need to change the items the NPC is selling.
This will be the real issue.

Send me the NPC entry for the vendor and the items that it has missing or wrongly selling.

**Comment by Dranah** [comment]
Well, the custom stuff isn't my basis for testing, nor it should be for anyone else.

NPC ID=34252 is the regular vendor, and Emalon the Storm watcher (NPC ID=33993) is dropping the hand/leg pieces. These are currently the ones with faulty items, but it's not the IDs that are actually wrong, but the stats/bonuses.

Let me explain. There are two sets with identical names in the game atm, both titled as "Conqueror's Darkruned Plate" which is the DK tank t8 name.
The item set you listed ( 46238, 46233, 46234, 46236, 46240) is however not meant to be the tank t8. On wowhead data, it is a transmog set called "Darkruned Plate" such as this https://www.wowhead.com/item=46240/darkruned-pauldrons and it has different color from the original set.
This set shouldn't exist or be available to players and should not be on your custom t8 vendor either, because it is not the actual t8 set.
However, this set currently has the proper stats and set bonuses.

The other set is the intended DK tank t8 of 3.3.5, with item IDs 46118,46119,46120,46121,46122.
This set however had its stats and bonuses broken for some reason, and you have been fixing them along with my previous reports, but hasn't finished the job yet.
This set has the correct t8 color, this set is the one that needs to be on the vendor and emalon as well, however their stats and set bonuses need to be fixed because they are currently fucked up.

TL;DR
**How it should work:**
The vendors (including the mall one) need to be selling item IDs 46118,46119,46120,46121,46122
HOWEVER, the above set needs to get their stats and bonuses fixed to match those on set 46238, 46233, 46234, 46236, 46240)

**Comment by Dranah** [comment]
Item set 46118,46119,46120,46121,46122 had its stats fixed.
Now this set needs to be sold by npc=34252, npc=70022, dropped by npc=33993. In place of the other set with different colour.

**Comment by Dranah** [comment]
This last change will fix the color of the set as well.

**Comment by Dranah** [comment]
The correct set is sold by both penny maniac and the dalaran vendor now.
1 Issue remains: The dalaran vendor now sells both copies of the leggings. ID 46238 needs to be removed from vendor ID 34252.
